src/app.ts

```diff
@@ -19,3 +20,13 @@ window.mediaElement = document.getElementById('video-player');
 window.playlist = [];
 window.currentPlaylistIndex = -1;
 window.repeatMode = RepeatMode.RepeatNone;
+
+window.i18n = i18n;
+
+(document.getElementById(
```

Comment:
DocumentManager could handle this post-pr :)

Reply:
Maybe with a updateLocale function that we can call if we have to change the locale while the app is running?